.github/ISSUE_TEMPLATE/1_bug_report_form.yml

```diff
@@ -17,6 +17,12 @@ body:
       description: A concise description of what issue you're experiencing. **Example:** "When I click on the 'Sign in' button, I expect to be redirected to the login page, but instead I get a 404 error."
     validations:
       required: true
+  - type: textarea
```

Comment:
Please change the type to `input`

Reply:
Author might not take note seriously and may skip this process,
most of the author do not remain active for a long time so if someone tries to ask the author to provide more steps to reproduce they will not respond for long time and this will be helpful so that anyone can check the reproducibility of the issue just by clicking on the link.
so i think the text area will be the best.

Reply:
Please change the type to `input`

Reply:
Done